resources/rtos.css

```diff
@@ -1,6 +1,6 @@
 .threads-grid {
   gap: 0;
-  text-align: right;
+  text-align: center;
```

Comment:
I am not comfortable with changing the default to 'center'. I want all numeric fields at least to be right-justified. Center justified looks weird. We should figure out which columns should be centered.

Reply:
As most numbers are in hex or have the same/similar number of digits, i personally do not see a big difference in having it center aligned.
Also i'm not used to see right aligned stuff a lot (here in Germany).
Also the new percentage view eliminated a lot of numbers for a much better visual appearance (way to much "meaningless" address entries which needed manual calculations before) at least for the uC/OS-II implementation.
But should be easy with css to set the alignment per column.

My preferences are:
- Text columns should be either center or left aligned

- Percentage columns are center aligned

- not sure if Priority is a numeric value for you as in FreeRTOS it has a second value behind it. But no clear preferences from my side here

- id also not a clear preference on that

One thing to test here is, if mixing alignment in a table really looks good and allows easy reading of all elements.